### PR TITLE
Add paydown timeline item to transaction details

### LINKED ISSUE #3588: Add loan paydown to transaction timeline
The transaction details page should indicate if there's a loan paydown related to the transaction with a new timeline item. The loan link should send the user to the transactions page with the loan ID filtered. 

image

### PR DESCRIPTION
**Pull request by tpaksu**
Fixes #3588

#### Changes proposed in this Pull Request

This PR will add a timeline item showing the paydown transaction in the transaction details page for the charges having a portion taken back as a loan payment. Also it'll show the repayment amount in the transaction details overview card and subtract the amount from the net amount displayed there.

This depends on 1583-gh-Automattic/woocommerce-payments-server to get the paydown event into the timeline. So be sure you've checked out on your server (or it is merged already) before you start testing this.

The timeline item will be shown like this: 

image

Updated parts: 

- The header has "Loan repayment" node, which is subtracted from the net amount.
- There's a new timeline item showing the loan repayment, with a link to the loan filtered transactions page

#### Testing instructions

* All tests should pass
* Check if the overview has the "Loan repayment: ###" text showing the amount of the repayment on the transaction summary card above.
* Check if you see the timeline item, showing the correct values
* Check if you don't see them both on transactions without loan repayments.
* Check if the net amount on the transaction summary card shows the net amount with the loan repayment subtracted.

-------------------

- [x] Added changelog entries to both `readme.txt` and `changelog.txt` (or does not apply)
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) : _Add link here / 'QA Testing Not Applicable'_

### CODE REVIEW

**Comment by tpaksu** [comment]
I set this to on hold, because it depends on previous PR's to get merged first.

**Review by luizreis — COMMENTED** [review]
Thanks @tpaksu - as clarified on slack (p1644500639371839/1644489758.972789-slack-C01B8KNUYSW), there are a few intentional differences with the layout on Figma.

This is looking pretty good! I tested the changes and they work well for charges that have a paydown and charges that don't. The new UI also doesn't show up when the capital feature flag is disabled.

Before we merge this, do you think we should include some tests with these changes?

**Review by luizreis — APPROVED** [review]
Great work, @tpaksu! Thanks for adding translation to that string and adding some tests. :shipit:

**Pull request merged**
